pywincffi/kernel32/file.py

```diff
@@ -27,7 +29,8 @@ def CreateFile(  # pylint: disable=too-many-arguments
         https://msdn.microsoft.com/en-us/library/aa363858
         https://msdn.microsoft.com/en-us/library/gg258116
 
-    :param str lpFileName:
+    :param unicode/str lpFileName:
+        Type is unicode on Python 2, str on Python 3.
```

Comment:
I'd rephrase so it's clear the doc string is talking about the objects:

```
Type is ``unicode`` on Python 2, ``str`` on Python 3.
```

You could also do:

```
Type is :class:`unicode` on Python 2, :class:`str` on Python 3.
```

I'd suggest the first version though because it render identically in Python 2 and 3 (docs could be built in either version).